extensions/wikia/AdEngine/js/provider/gpt/googleTag.js

```diff
@@ -140,5 +142,12 @@ define('ext.wikia.adEngine.provider.gpt.googleTag', [
 		registeredCallbacks[id] = callback;
 	};
 
+	GoogleTag.prototype.onAdLoad = function (slotName, element, gptEvent, onAdLoadCallback) {
```

Comment:
OK, nevermind. We did it before in `GptHelper`. So, it doesn't change anything ;) Correct me if I'm wrong, please :P